datumaro/plugins/openvino_plugin/launcher.py

```diff
@@ -82,15 +91,52 @@ def get_categories():
     def process_outputs(inputs, outputs):
         raise NotImplementedError("Function should be implemented in the interpreter script")
 
+    @staticmethod
+    def normalize(inputs):
+        raise NotImplementedError("Function should be implemented in the interpreter script")
+
 
 class OpenvinoLauncher(Launcher):
     cli_plugin = _OpenvinoImporter
 
     def __init__(
-        self, description, weights, interpreter, device=None, model_dir=None, output_layers=None
+        self,
+        description=None,
+        weights=None,
+        interpreter=None,
+        model_dir=None,
+        model_name=None,
+        output_layers=None,
+        device=None,
     ):
+        if model_name:
+            model_dir = os.path.join(os.path.expanduser("~"), ".cache", "datumaro")
+            if not osp.exists(model_dir):
+                os.makedirs(model_dir)
+
+            # Please visit open-model-zoo repository for OpenVINO public models if you are interested in
+            # https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/index.md
+            url_folder = "https://storage.openvinotoolkit.org/repositories/datumaro/models/"
+
+            description = osp.join(model_dir, model_name + ".xml")
+            if not osp.exists(description):
+                cached_description_url = osp.join(url_folder, model_name + ".xml")
+                log.info('Downloading: "{}" to {}\n'.format(cached_description_url, description))
+                self._download_file(cached_description_url, description)
+
+            weights = osp.join(model_dir, model_name + ".bin")
+            if not osp.exists(weights):
+                cached_weights_url = osp.join(url_folder, model_name + ".bin")
+                log.info('Downloading: "{}" to {}\n'.format(cached_weights_url, weights))
+                self._download_file(cached_weights_url, weights)
+
+            if not interpreter:
+                openvino_plugin_samples_dir = get_samples_path()
+                interpreter = osp.join(openvino_plugin_samples_dir, model_name + "_interp.py")
```

Comment:
I think that it is more appropriate to move the model download functionality to `_OpenvinoImporter` but leave it in this PR.